ci-operator/config/openshift/cluster-api/openshift-cluster-api-release-4.0.yaml

```diff
@@ -1,8 +1,13 @@
+tag_specification:
+  name: '4.0'
+  namespace: ocp
+promotion:
+  name: '4.0'
+  namespace: ocp
 base_images:
   base:
-    cluster: https://api.ci.openshift.org
```

Comment:
Why are we losing this?

Reply:
This only works safely unprotected images - there is a ci-operator bug / complexity where if you specify this and your KUBECONFIG points to anything different it'll error out.  I want to fix it at some point but it's tough to do now.

https://github.com/openshift/ci-operator/blob/106a50e1c66f487147758f991cf2f21026abe72b/pkg/defaults/defaults.go#L109

can't correctly normalize all possible URLs to determine if they're the same.  Also, when ci-operator is running on cluster it talks  to the service IP (`170.30.0.1`) which can't be trivially checked against equality.